semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/package.scala

```diff
@@ -188,4 +187,22 @@ package object language {
   implicit def toCallForCallGraph(steps: Steps[nodes.Call]): Call =
     new Call(steps.raw)
 
+  // Locations
+
+  implicit def toExtendedNode(node: Node): ExtendedNode =
```

Comment:
I’m not sure if this was directly copied over but an `implicit class` would be a much better fit here.